spec/factories/facility_groups.rb

```diff
@@ -13,17 +13,11 @@
     protocol
 
     transient do
-      create_parent_region { Flipper.enabled?(:regions_prep) }
+      create_parent_region { true }
     end
 
     before(:create) do |facility_group, options|
       if options.create_parent_region
-        if facility_group.organization.region.nil?
```

Comment:
This should never happen anymore afaict. Everything should have a region for itself. In tests, both FGs and Facilities should always have a parent region created as well (unless specified otherwise).